src/environments/unfederated_environment.c

```diff
@@ -37,6 +37,14 @@ static lf_ret_t Environment_wait_until_locked(Environment *self, instant_t wakeu
   }
 }
 
+static lf_ret_t Environment_wait_for(Environment *self, interval_t wait_time) {
```

Comment:
Maybe some docs string would be nice, because this is technically user-facing api.

Reply:
The doc strings are in the header file